app/satellite/middleware.ts

```diff
@@ -0,0 +1,33 @@
+import type express from 'express';
```

Comment:
I think we should consider removing this, and using Remix's error handling.  It has a whole bunch of stuff we can leverage.